ipn/ipnstate/ipnstate.go

```diff
@@ -111,12 +112,12 @@ type PeerStatus struct {
 
 	// Tags are the list of ACL tags applied to this node.
 	// See tailscale.com/tailcfg#Node.Tags for more information.
-	Tags []string `json:",omitempty"`
+	Tags views.StringSlice
```

Comment:
To get omitempty working, you'll probably need to make these pointer-to-view

Reply:
Done